auth-server/src/services/email.ts

```diff
@@ -177,3 +177,34 @@ export const sendProposalCreatedEmail = (user: User, link: string) => {
 	sgMail.send(msg).catch(e =>
 		console.error('Proposal Created Email not sent', e));
 };
+
+export const sendNewProposalCreatedEmail = (user: User, type: string, postId: number) => {
+	if (!apiKey) {
+		console.warn('New Proposal Created Email not sent due to missing API key');
+		return;
+	}
+
+	const text = `
+		<p>
+			Hi ${user.name || ''}!<br/><br/>
+
+			There is a new ${type} on chain.<br />
+			Click on the following link to check it out: <a href="${DOMAIN}/${type}/${postId}">${DOMAIN}/${type}/${postId}</a>.<br /><br />
```

Comment:
referendum/postId will not work in most cases. It should be referendum/referendum_id